addons/godot_pixelorama_importer/animation_player_import.gd

```diff
@@ -0,0 +1,197 @@
+tool
+extends EditorImportPlugin
+
+const VISIBLE_NAME := "Sprite & AnimationPlayer"
+
+var editor: EditorInterface
+
+
+func _init(editor_interface):
+	editor = editor_interface
+
+
+func get_importer_name():
+	return "com.technohacker.pixelorama.animationplayer"
+
+
+func get_visible_name():
+	return VISIBLE_NAME
+
+
+func get_recognized_extensions():
+	return ["pxo"]
+
+
+# We save directly to stex because ImageTexture doesn't work for some reason
+func get_save_extension():
+	return "tscn"
+
+
+func get_resource_type():
+	return "PackedScene"
+
+
+func get_import_options(_preset):
+	var default_scale: Vector2 = ProjectSettings.get_setting("pixelorama/default_scale")
+	var default_external_save: bool = ProjectSettings.get_setting(
+		"pixelorama/default_animation_external_save"
+	)
+	var default_external_save_path: String = ProjectSettings.get_setting(
+		"pixelorama/default_animation_external_save_path"
+	)
+
+	return [
+		{"name": "Sprite2D", "default_value": false, "usage": PROPERTY_USAGE_GROUP},
+		{"name": "scale", "default_value": default_scale},
+		{"name": "Animation", "default_value": false, "usage": PROPERTY_USAGE_GROUP},
+		# 65536 = PROPERTY_USAGE_UPDATE_ALL_IF_MODIFIED, but not exported in GDscript.
+		{
+			"name": "external_save",
+			"default_value": default_external_save,
+			"usage": PROPERTY_USAGE_DEFAULT | 65536
```

Comment:
Maybe `(1 << 16)` would be more clear for this purpose, but I'm not too sure :thinking:

Reply:
Yeah, it's better with 1 << 16, didn't know it worked in gdscript